docs/Security.rst

```diff
@@ -658,6 +661,45 @@ configuration :
             ...
 
 
+.. _Security.SameSiteConfiguration:
+
+======================
+SameSite Configuration
+======================
+
+SameSite is a property set on HTTP cookies. It can prevent some CSRF attacks.
+SameSite property can take one of theses values : None, Strict, and Lax
+
+With value **None**, when a request is done on eXo Server, there is no verification on the referer. The cookies is used.
+For example, if a user receive a forged email, containing a link to call a REST allowing to modify data, like delete a space,
+if the user have a session on eXo, when he clicks on the link, the session cookie is used and the request is executed. It is a
+CSRF attack.
+
+With value **Strict**; when a request is done on eXo Server, the referer is checked. If the referer has a different
+domain than the eXo Server domain, the request will not use the cookie.
+In the above example, the request is not directly executed, and the user is redirected to the login URL.
+This behaviour is also applied for HTTP GET requests. So, when a user clicks on a link in a notification for example, he has to
+login again. 
+With this value, all SSO systems (SAML, OAuth, OpenIdConnect ...), generally based on redirections between different hosts, **will not work**.
+
+With value **Lax**; when a request is done on eXo Server, the referer is checked. If the referer has a different
+domain than the eXo Server domain, and if the request is not a GET request, the request will not use the cookie. So this
+intermediate option allows to use link in mail notification, and protects sensitive urls which modify data.
+
+By default, eXo use **Lax** value for cookies.
```

Comment:
```suggestion
By default, eXo uses **Lax** policy in order to have email notification links work on SSO contexts.
```

Reply:
Links in email and SSO contexts are not related. 
With Strict mode, links in notification will request for a new login. And the SSO will not work.
With Lax mode, links in notification will correctly open the page. And SSO will work.

```suggestion
By default, eXo uses **Lax** policy in order to have email notification links and SSO configurations work.
```

WDYT ?